src/main/java/co/elastic/support/monitoring/MonitoringExportService.java

```diff
@@ -129,7 +129,7 @@ public void execExtract(MonitoringExportInputs inputs) throws DiagnosticExceptio
             logger.error( "Unexpected error occurred", t);
             logger.error(Constants.CONSOLE, "Unexpected error. {}", Constants.CHECK_LOG);
         } finally {
-            ResourceCache.textIO.dispose();
+            // resourceCache.textIO.dispose();
```

Comment:
Should this not be "disposed" of?

```suggestion
            resourceCache.textIO.dispose();
```

or

```suggestion
```